ui/raidboss/data/03-hw/dungeon/fractal_continuum.txt

```diff
@@ -0,0 +1,156 @@
+hideall "--reset--"
+hideall "--sync--"
+hideall "--start--"
+
+0 "--reset--" sync /:Exhibit level III is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The high-level incubation bay is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The reality augmentation bay is no longer sealed/ window 10000 jump 0
+
+
+###Phantom Ray
+#-ii F7D
```

Comment:
-ii F7D -p F7F:100

Reply:
Can you clarify that? Overclock is included in the timeline, I simply changed the display to "Damage Up" (or the equivalent localization) since there's never a direct cast and you never see an on-screen display as you do with the instant-cast tankbusters/AoEs.

Reply:
Sorry, I was just suggesting that if you have the second phase starting at time=100, that you add that in to the args as well.  `-p` is the phase arg, so `-p F7F:100` says take the first instance of skilll `F7F` and adjust it to be time=100.

Reply:
Oh, I see. Yes, you're right, that does make sense.